pkg/transformers/labelsandannotationsconfig.go

```diff
@@ -95,12 +95,7 @@ var defaultLabelsPathConfigs = []PathConfig{
 	{
 		GroupVersionKind:   &schema.GroupVersionKind{Group: "batch", Kind: "CronJob"},
 		Path:               []string{"spec", "jobTemplate", "spec", "selector", "matchLabels"},
-		CreateIfNotPresent: true,
-	},
-	{
-		GroupVersionKind:   &schema.GroupVersionKind{Group: "batch", Kind: "CronJob"},
-		Path:               []string{"spec", "jobTemplate", "spec", "metadata", "labels"},
-		CreateIfNotPresent: true,
+		CreateIfNotPresent: false,
```

Comment:
why do you need to remove the pathconfig for `[]string{"spec", "jobTemplate", "spec", "metadata", "labels"},` in cronjob? Does it work by changing `CreateIfNotPresent` to `false`?

Reply:
It will, but only because then it won't create it. That `spec.jobTemplate.spec.metadata` isn't valid for a CronJob at all from what I can tell. Literally anything being set there causes K8s to complain--that's the core of #116. `spec.template.metadata.labels` seems to be the correct place to set that information (and it's already in the transform).

Reply:
I see, it is an invalid path. Thank you for your explanation.